README.md

```diff
@@ -1,180 +1,30 @@
 # Microsoft Azure SDK for .NET
 
-| Component | Build Status |
-| --------- | ------------ |
-| Management Libraries | [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-net.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-net) |
-| Client Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/41?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=41&branchName=master) |
+This repo contains official .NET client libraries for Microsoft Azure services.
 
-Microsoft Azure is an ever-expanding set of cloud services to help your organization meet your business challenges. It’s the freedom to build, manage, and deploy applications on a massive, global network using your favorite tools and frameworks.
+## NuGet Packages
 
-This repo contains the libraries to allow you to easily leverage Azure from your applications and tools.
+[All Nuget Packages](https://github.com/Azure/azure-sdk-for-net/tree/master/Documentation/sdk-for-net-packages.md).
```

Comment:
We now have a package index at https://github.com/Azure/azure-sdk-for-net/blob/master/packages.md. @scbedd we should probably figure out if we can remove Documentation/sdk-for-net-packages.md and perhaps we can also update our generated index to have the nice nuget badges.

Reply:
Sounds good. I will updated this PR.